guide/14-deep-learning/how_AutoDL_works.ipynb

```diff
@@ -0,0 +1,580 @@
+{
```

Comment:
<p>The output of this function will be a list of pixel classification models supported by the <code>AutoDL</code> class.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1198/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>